Osiris/SDK/Entity.h

```diff
@@ -235,7 +235,33 @@ class Entity {
         callVirtualMethod<void>(this, 345, std::ref(vec));
         return vec;
     }
+    
+    bool isInThrow() noexcept {
+        auto localPlayer = interfaces.entityList->getEntity(interfaces.engine->getLocalPlayer());
+        if (!localPlayer || localPlayer->isAlive())
+            return false;
+
+        auto activeWeapon = localPlayer->getActiveWeapon();
+        if (!activeWeapon)
+            return false;
+
+        auto weaponIndex = getWeaponClass(activeWeapon->itemDefinitionIndex2());
+        if (!weaponIndex)
+            return false;
+        if (weaponIndex == 40) {
```

Comment:
magic number, enum please?
you created the enum and everything, you just are not using it here*